api/v1/service/resource/resource.go

```diff
@@ -28,12 +28,15 @@ import (
 	"github.com/tektoncd/hub/api/pkg/db/model"
 	res "github.com/tektoncd/hub/api/pkg/shared/resource"
 	"github.com/tektoncd/hub/api/v1/gen/resource"
+	"gorm.io/gorm"
 )
 
 type service struct {
 	app.Service
 }
 
+const Latest = "latest"
```

Comment:
why are we exporting this variable? can we keep it as `const latest = "latest"` ?